egs/ljspeech/asr1/RESULTS.md

```diff
@@ -0,0 +1,18 @@
+<!-- Generated by ../../../utils/show_result.sh -->
+# RESULTS
+## Environments
+- date: `Thu Jun  4 10:58:46 JST 2020`
+- python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
+- espnet version: `espnet 0.7.0`
+- chainer version: `chainer 6.0.0`
+- pytorch version: `pytorch 1.0.1.post2`
+- Git hash: `4e2079ebad5dcf2945c615c63165cd9e80959593`
+  - Commit date: `Thu Jun 4 10:57:36 2020 +0900`
+
+## char_train_no_dev_pytorch_train_mtlalpha1.0
```

Comment:
Will you have other results?
I saw you put another decoding configuration.

Also, it is great if you also upload your model.
Again, please send me (shinjiw@ieee.org) your email directly so that I'll make access to our Google drive.

Reply:
I'm still calculating something, so I'll add the results to RESULTS.md when it comes out and add it to Push.

Thank you. I'll send you an email.